app/views/catalog/email_success.html.erb

```diff
@@ -1,6 +1,5 @@
-<%= render Blacklight::System::ModalComponent.new do |component| %>
```

Comment:
Is this effectively requiring browsers support javascript?

Reply:
The `_success` template is only rendered when the request is initiated by Javascript (being an XHR request), right? https://github.com/projectblacklight/blacklight/pull/2573/files#diff-20dfe95ebbfa7c11f6eff4bead05d3a0b0977b382a0164757c80f29cf5a02061R41